debug.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+	cat <<USAGE
+Usage: $0 BROWSER JENKINS [ARGS]
+
+The script runs dryrun tests first to discover trivial problems immediately.
+It can use jenkins.war from local maven repository or download it when missing.
+
+BROWSER: Value for BROWSER variable
+JENKINS: Path to the jenkins.war, Jenkins version of one of "latest", "latest-rc", "lts" and "lts-rc"
+
+Examples:
+
+# Run full suite in FF against ./jenkins.war.
+$ ./run firefox ./jenkins.war
+
+# Run Ant plugin test in chrome against Jenkins 1.512.
+$ ./run chrome 1.512 -Dtest=AntPluginTest
+
+# Run full suite in FF against LTS release candidate
+$ ./run firefox lts-rc
+USAGE
+  exit -2
+fi
+
+browser=$1
+war=$2
+if [ ! -f $war ]; then
+    mirrors=http://mirrors.jenkins-ci.org
+    case "$war" in
+        "latest")
+            war=jenkins-latest.war
+            url=$mirrors/war/latest/jenkins.war
+        ;;
+        "latest-rc")
+            war=jenkins-latest-rc.war
+            url=$mirrors/war-rc/latest/jenkins.war
+        ;;
+        "lts")
+            war=jenkins-lts.war
+            url=$mirrors/war-stable/latest/jenkins.war
+        ;;
+        "lts-rc")
+            war=jenkins-lts-rc.war
+            url=$mirrors/war-stable-rc/latest/jenkins.war
+        ;;
+    esac
+
+    if [ -n "$url" ]; then
+        find $war -maxdepth 0 -mtime +1 -delete 2> /dev/null
```

Comment:
`$war` may come from user parameters and hence may contain special symbols (like spaces in Windows paths). It would be better to escape this variable by using `"${war}"` here and below